src/components/DocumentInfo/AddressMultiple/AddressMultiple.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks";
```

Comment:
I think I would create this story differently

1. I would add a description of the different use cases that may happen
1. I would create a story for each scenario

The problem with the current approach is that it implies that you now what's going on, However storybook supposed to help you to understand the component and it's not the case

Reply:
ok created the respective story for each scenario